src/main/java/domain/lotto/Application.java

```diff
@@ -0,0 +1,39 @@
+package domain.lotto;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+    public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        System.out.println("구입금액을 입력해 주세요.");
+        double price = in.nextDouble();
+        System.out.println((int) (price / 1000) + "개를 구매했습니다.");
+        Lotto lotto = new Lotto(price);
+        for (List<Integer> i : lotto.allLotto.lottoPapers) {
+            System.out.println(i);
+        }
+        System.out.println("지난 주 당첨 번호를 입력해 주세요.");
+        in.nextLine();
+        String winning_number = in.nextLine();
+        System.out.println("보너스 볼을 입력해 주세요.");
+        int bonus_number = in.nextInt();
```

Comment:
```suggestion
        int bonusNumber = in.nextInt();
```
구글 자바 스타일 가이드에서 네이밍 정의는 카멜케이스로 되어 있습니다!

https://google.github.io/styleguide/javaguide.html#s5.3-camel-case